### PR TITLE
[Issue #13] Rocketmq-Dashboard License

### PR DESCRIPTION
**Pull request by StyleTang**
Revert the front end resource license modification (c9d4cd1b727 / 635e0ce4d04) and exclude the front end folder from apache-rat-plugin.
(front end resource license can be found in LICENSE file)

## What is the purpose of the change

update Rocketmq-Dashboard License

## Brief changelog

update Rocketmq-Dashboard License

## Verifying this change

Manual test passed

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -357,38 +357,16 @@ SOFTWARE.
...
-ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
-(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
-SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-
-The views and conclusions contained in the software and documentation are those
-of the authors and should not be interpreted as representing official policies,
-either expressed or implied, of the FreeBSD Project.
+https://github.com/apache/echarts
+
+                                 Apache License
```
Comment (vongosling) [inline LICENSE:362]: Good catch:-0

File: LICENSE [hunk 2]
```diff
@@ -413,17 +391,13 @@ Font-Awesome
...
-http://scripts.sil.org/OFL
-Font Awesome CSS, LESS, and Sass files are licensed under the MIT License:
-https://opensource.org/licenses/mit-license.html
-The Font Awesome documentation is licensed under the CC BY 3.0 License:
-http://creativecommons.org/licenses/by/3.0/
-Attribution is no longer required as of Font Awesome 3.0, but much appreciated:
-Font Awesome by Dave Gandy - http://fontawesome.io
-Full details: http://fontawesome.io/license/
+Font Awesome Free License
+Font Awesome Free is free, open source, and GPL friendly. You can use it for
```
Comment (vongosling) [inline LICENSE:395]: less is more. Could we just emphasize copyright here?

**Comment by vongosling** [comment]
@duhenglucky @SteNicholas Would you like to follow the best practices here to optimize the flink and Stream sub-projects :-) I could help to review.

File: LICENSE (hunk 2 above)
Comment (StyleTang) [reply LICENSE:395]: The license info is copied from https://fontawesome.com/license/free

**Pull request merged**